src/modules/tests/rekey-reason/rekey-reason.selector.ts

```diff
@@ -0,0 +1,15 @@
+import { RekeyReason, StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
+
+export const getReasonForRekey = (test: StandardCarTestCATBSchema) => {
+  return test.rekeyReason;
+};
+
+export const getRekeyIpadIssueSelected = (rekeyReason: RekeyReason): boolean => rekeyReason.ipadIssue.selected;
```

Comment:
You went slightly too granular with these selectors. Would create only 3: `getIpadIssue`, `getTransfer` and `getOther`.

Omitting `Rekey` should be the way to go because the file name already has the word `rekey` init